recipes/xbyak/all/conanfile.py

```diff
@@ -30,5 +30,4 @@ def package(self):
         self.copy("*.h", dst=os.path.join("include", "xbyak"), src=os.path.join(self._source_subfolder, "xbyak"))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "xbyak"
-        self.cpp_info.names["cmake_find_package_multi"] = "xbyak"
+        self.cpp_info.set_property("cmake_file_name", "xbyak")
```

Comment:
According to conan team, `.names` and `set_property()` should live together for the moment.

Reply:
@SpaceIm 
Thank you for the information.
I missed your comment and modified the other recipes already. 
I will fix those with the highest priority.

Reply:
Fixed